README.md

```diff
@@ -4,18 +4,30 @@
 * Your coffee beans now enter a "roasting" process where they are submitted to external elements, much like roasting coffee beans to bring out their best flavor. During this process, your labmates and collaborators can comment on the open GitHub issue to share their thoughts, raise concerns, etc.
 * The last stage in the process is "brewing" a quaffable cup of coffee. Here, you refine your original idea in light of the group discussion.
 * Finally, you and the PI enjoy the coffee :coffee: together (figuratively-speaking) as you formally pitch the idea to the PI as a prospective undertaking for the lab.
 
 ### How-To
+
+**Budding**
+
 1. Use one of the issue templates available in this repo (there is one for papers and one for code/tools/instruments) to open a new issue. Take your time in formalizing your thoughts. While you work on the issue, it will live in the "Budding" pipeline on the brainBox ZenHub workflow.
-2. If you need or want to store specific files with your idea (such as diagrams), create a new branch within the brainBox repo. Name it: `[project shortname]-[your name]`. For example: `choco-factory-charlie`. Add the new directory under the `brainy-ideas` directory. The project shortname that you use for the GitHub issue, the shortname that you use for the new branch, and the shortname that you use for the directory within the repo should be identical.
-3. When you are ready, move the issue into the "Roasting" pipeline; open your idea up for group discussion by tagging the lab.
-4. Watch and respond to the questions that come in. Feel free to remind certain individuals to comment with their thoughts (by tagging them individually) if you feel they would have particularly valuable expertise to hone your idea.
-5. As your idea takes full shape, move the issue into the "Brewing" pipeline. Craft a clear pitch for the PI's approval.
-6. Use Calendly to pre-set a meeting with the PI, at least two days after you intend to submit the pull request on GitHub.
-7. Submit the pull request on GitHub by adjusting the details on the default pull request form.
-8. Get excited to discuss how to make your idea a reality!
+
+**Roasting**
+
+2. Create a new branch within the brainBox repo. Name it: `[project shortname]-[your name]`. For example: `choco-factory-charlie`. Add a new directory under the `brainy-ideas` directory. The project shortname that you use for the GitHub issue, the shortname that you use for the new branch, and the shortname that you use for the sub-directory should be identical.
+3. On your branch, within your project subfolder in the `brainy-ideas` directory, create a readme file. Use this file to build out and refine your idea.  This can start as a duplication of the content of your issue. 
+4. When you are ready, move the issue into the "Roasting" pipeline; open your idea up for group discussion by tagging the lab (@labwide) on the issue.
```

Comment:
@jessb0t is "moving to the roasting pipeline" a conceptual thing, a move in zenhub, or something else? 
please make explicit